strawberry/schema/config.py

```diff
@@ -9,6 +9,7 @@
 class StrawberryConfig:
     auto_camel_case: InitVar[bool] = None  # type: ignore
     name_converter: NameConverter = field(default_factory=NameConverter)
+    description_from_docstrings: bool = False
```

Comment:
@BryceBeagle what do you think of this name? 😊

Reply:
I think `descriptions_from_docstrings` would be better. Or maybe even just `docstring_descriptions`. I'm wondering if it could be useful to do something like:

```python
from enum import auto, Flag

class DocstringDescriptions(Flag):
    NONE = auto()
    TYPES = auto()
    RESOLVERS = auto()
    ARGUMENTS = auto()
    ALL = TYPES | RESOLVERS | ARGUMENTS

@dataclass    
class StrawberryConfig:
    docstring_descriptions: DocstringDescriptions = DocstringDescriptions.NONE
```

This would allow users to have a little more granularity with which docstrings are getting enabled